site/en/docs/extensions/mv3/content_scripts/index.md

```diff
@@ -380,20 +380,20 @@ window.setTimeout(function() {
 
 [1]: http://www.w3.org/TR/DOM-Level-2-HTML/
 [2]: /docs/extensions/mv3/messaging
-[3]: /docs/extensions/i18n
-[4]: /docs/extensions/storage
-[5]: /docs/extensions/runtime
-[6]: /docs/extensions/runtime#method-connect
-[7]: /docs/extensions/runtime#method-getManifest
-[8]: /docs/extensions/runtime#method-getURL
-[9]: /docs/extensions/runtime#property-id
-[10]: /docs/extensions/runtime#event-onConnect
-[11]: /docs/extensions/runtime#event-onMessage
-[12]: /docs/extensions/runtime#method-sendMessage
+[3]: /docs/extensions/reference/i18n
+[4]: /docs/extensions/reference/storage
+[5]: /docs/extensions/reference/runtime
+[6]: /docs/extensions/reference/runtime#method-connect
+[7]: /docs/extensions/reference/runtime#method-getManifest
+[8]: /docs/extensions/reference/runtime#method-getURL
+[9]: /docs/extensions/reference/runtime#property-id
+[10]: /docs/extensions/reference/runtime#event-onConnect
+[11]: /docs/extensions/reference/runtime#event-onMessage
+[12]: /docs/extensions/reference/runtime#method-sendMessage
 [13]: #programmatic
 [14]: /docs/extensions/declaratively
```

Comment:
```suggestion
[14]: #declaratively
```

The currently target does not exist. This should reference another section in the same page.